src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person}) => {
+  
+  const { name, age, sex, isMarried, partnerName } = person;
+  return (
+  <section className = "Person">
+    <h2 className="Person__name">My name is {name}</h2>
+    {age !== undefined && (
+    <p className="Person__age">I am {age}</p>
+    )}
```

Comment:
```suggestion
    {age && (
      <p className="Person__age">I am {age}</p>
    )}
```
You don't need to compare with an undefined, https://developer.mozilla.org/en-US/docs/Glossary/Truthy